src/get-started/first-java-service.md

```diff
@@ -0,0 +1,615 @@
+```
+
+When you see messages like the following, the network is active:
+
+<!-- markdownlint-disable line-length -->
+```text
+[2020-03-05T10:36:24Z INFO  exonum_node::consensus] COMMIT ====== height=4, proposer=0, round=1, committed=0, pool=0, hash=43ac20f8b...
+```
+<!-- markdownlint-enable line-length -->
+
+Open a separate shell session and check the active services:
+
+```shell
+# You may pipe the response into `jq` to pretty-print it, if you have it
+# installed:
+curl -s http://127.0.0.1:3000/api/system/v1/services # | jq
+```
+
+You can see in the output the lists of deployed _service artifacts_ and
+_service instances_.
+However, the network has neither our _service artifact_ nor
+an _instance_ of our service. That is natural, because the service
+must be registered in the network first, and then it may be instantiated.
+
+### Service Instantiation
+
+#### Install the Java Launcher
+
+To register a service artifact that we built previously in the network,
+we will need `exonum-launcher` tool. It is a Python application which we
+recommend to install in a [virtual environment][python-venv]:
+
+```shell
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+[python-venv]: https://docs.python.org/3/library/venv.html
+
+Then install the `exonum-launcher` with the Java runtime support:
+
+```shell
+pip install exonum-launcher-java-plugins
+```
+
+Check it works:
+
+```shell
+python -m exonum_launcher --help
+```
+
+#### Start a Test Instance
+
+Next, we shall place the service artifact into an artifacts directory
+of the node: `testnet/artifacts`.
+
+```shell
+# Create the artifacts directory
+mkdir testnet/artifacts
+# Copy the artifact
+cp car-registry-service/target/car-registry-service-1.0.0-SNAPSHOT-artifact.jar \
+   testnet/artifacts/
+```
+
+<!--
+TODO: Shall the _node_ (= Java runtime) create an artifacts directory if one
+does not exist already? If it shall, won't it cause problems if we launch
+several nodes locally with the same (not yet existing) artifacts directory?
+
+It is somewhat annoying to always have to create the dir :-)
+-->
+
+<!--
+TODO: Shall we place the burden of copying (= uploading the JAR)
+on the launcher-plugin + Java runtime pair?
+-->
+
+Launch the service:
+
+```shell
+python -m exonum_launcher -i deploy-start-config.yml
+```
+
+Launcher will take the service instance name and other parameters from
+the configuration file, and submit the request to the node.
+The launcher must print the status of the service artifact deploy
+and the service instance start.
+We can also verify that both operations succeeded via the node API:
+
+```shell
+curl -s http://127.0.0.1:3000/api/system/v1/services | jq
+```
+
+#### Invoke the Service Operations
+
+We will use a light client application to invoke the service operations.
+Development of service client applications is not covered in this tutorial,
+but the client for the car registry is provided in the tutorial
+repository as a third module, `car-registry-client`.
```

Comment:
a few words why do we need the client?

Reply:
> to invoke the service operations.

Isn't that enough (I assume readers know that by operations we consider transactions and read requests)?